plugins/modules/mso_schema_template_anp_epg.py

```diff
@@ -334,7 +377,9 @@ def main():
 
     epgs_path = '/templates/{0}/anps/{1}/epgs'.format(template, anp)
     epg_path = '/templates/{0}/anps/{1}/epgs/{2}'.format(template, anp, epg)
+    service_path = '/templates/{0}/anps/{1}/epgs/{2}/cloudServiceEpgConfig'.format(template, anp, epg)
```

Comment:
You could probably do:
service_path = '{0}/cloudServiceEpgConfig'.format(epg_path)